x-pack/plugins/ml/server/models/data_recognizer/modules/network/ml/datafeed_rare_destination_country.json

```diff
@@ -0,0 +1,25 @@
+{    "job_id": "rare-destination-country",
+   "indices": ["logs-*","filebeat-*","packetbeat-*"],
+    "max_empty_searches": 10,
+      "query": {
+      "bool": {
```

Comment:
same as above - this should be rewritten:

```
"bool": {
    "filter": [
        {"term": {"event.category": "network"}},
        {"exists": {"field": "destination.geo.country_name"}}
    ]
}
```

Reply:
done